Ref/Top/Main.cpp

```diff
@@ -11,7 +12,33 @@ void print_usage(const char* app) {
 #include <signal.h>
 #include <cstdio>
 
+class Logger :
```

Comment:
This is typically done by the following:

```
#include <Os/Log.hpp>

Os::Log logger;
```

This will use the Os::Log implementation as the logger.  No need to define a custom logger here.  My guess is this code was done in the files replace by the autocoder output.

Reply:
Good point. I will fix it.